.travis.yml

```diff
@@ -88,6 +88,8 @@ script:
         . scripts/get-libjvm-path.sh || return $?;
         export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$JNI_LIBDIR";
         make func-test V=1;
+        make VERBOSE=1 pytest-self-check;
```

Comment:
We could add a separate parallel job for `self-check` in the future. It does not depend on the compilation of syslog-ng.

Reply:
Yes, you are right. For now is it ok to leave it here?